prometheus/common/alert-rules.d/crunchy-alert-rules-patroni.yml.example

```diff
@@ -0,0 +1,99 @@
+###
+#
+# Copyright © 2017-2022 Crunchy Data Solutions, Inc. All Rights Reserved.
+#
+###
+
+groups:
+- name: alert-rules
+  rules:
+
+########## PATRONI RULES ##########
+
+  - alert: PatroniPGIsRunning
+    expr: patroni_postgres_running != 1
+    for: 60s
+    labels:
+      service: patroni 
+      severity: critical
+      severity_num: 300
+    annotations:
+      summary: 'Patroni is reporting that PostgreSQL is not running on {{ $labels.job }}'
+
+  - alert: PatroniPendingRestart
+    expr: patroni_pending_restart != 0
+    for: 1800s
+    labels:
+      service: patroni 
+      severity: warning
+      severity_num: 300
+    annotations:
+      summary: 'Patroni is reporting a pending restart on {{ $labels.job }}'
+
+  - alert: PatroniIsPaused
+    expr: patroni_is_paused != 0
+    for: 900s
+    labels:
+      service: patroni 
+      severity: critical
+      severity_num: 300
+    annotations:
+      summary: 'Patroni is reporting that it is in maintenance mode (paused) on {{ $labels.job }}'
+
+# Threshold value here should be based on Patroni timeout values to its DCS target
+  - alert: PatroniDCSLastSeen
+    expr: (time() - patroni_dcs_last_seen) > 300
+    for: 60s
+    labels:
+      service: patroni 
+      severity: warning 
+      severity_num: 200
+    annotations:
+      summary: 'Patroni running on {{ $labels.job }} has reported that it has not communicated with its DCS for at least 5 minutes'
+
+# Threshold value here should be based on Patroni timeout values to its DCS target
+  - alert: PatroniDCSLastSeen
+    expr: (time() - patroni_dcs_last_seen) > 1800
+    for: 60s
+    labels:
+      service: patroni 
+      severity: critical 
+      severity_num: 300
+    annotations:
+      summary: 'Patroni running on {{ $labels.job }} has reported that it has not communicated with its DCS for at least 30 minutes'
+
+# Detects a given amount of timeline switches in a given time period. Default example is 5 switches within an hour.
```

Comment:
```suggestion
# Detects a certain number of timeline switches in a given time period. Default example is 5 switches within an hour.
```

Reply:
I think the word "given" fits better hear since it intended as a user configured option. Unless they're happy with the default.

Reply:
Works for me.